lib/binance.ex

```diff
@@ -221,6 +221,18 @@ defmodule Binance do
     end
   end
 
+  @doc """
+  Fetches system status.
+
+  Returns `{:ok, %Binance.SystemStatus{}}` or `{:error, reason}`.
+  """
+  def get_system_status() do
+    case HTTPClient.get_binance("/wapi/v3/systemStatus.html") do
```

Comment:
As per my comment `wapi` endpoints are getting delisted but we could use `/sapi/v1/system/status`